src/test/java/org/apache/flink/benchmark/StreamNetworkThroughputBenchmarkExecutor.java

```diff
@@ -62,20 +63,37 @@ public void networkThroughput(MultiEnvironment context) throws Exception {
 	 */
 	@State(Thread)
 	public static class MultiEnvironment extends StreamNetworkThroughputBenchmark {
-		@Param({"1", "100"})
-		public int channels = 1;
 
-		@Param({"1", "4"})
-		public int writers = 1;
+		//Ideally we would like to run 1,100ms, 1000,1ms, 1000,100ms. However 1000,1ms is too slow to execute.
+		@Param({"1,100ms", "100,1ms", "1000,100ms"})
+		public String channelsFlushTimeout = "100,100ms";
 
-		@Param({"1", "100"})
-		public int flushTimeout = 1;
+		//Do not spam continuous benchmarking with number of writers parameter.
+		//@Param({"1", "4"})
+		public int writers = 4;
 
 		@Setup
 		public void setUp() throws Exception {
+			int channels = parseChannels(channelsFlushTimeout);
+			int flushTimeout = parseFlushTimeout(channelsFlushTimeout);
```

Comment:
At some point, we should do the parsing only once, but imho, this is ok for now.

Reply:
Yes, agree, I'm not proud of it, but I didn't want to over-engineer it at this moment by introducing additional pojo to return both values at once :(